tests/FakeItEasy.Tests/Core/SequentialCallContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Core;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SequentialCallContextTests

```

Comment:
Hear hear. I've come largely around to "if the acceptance tests cover the situation, they're good enough". Of course, exceptions should be made for very complicated logic or for conditions that are hard to simulate from the outside, but I don't think there's a need here.